frontend/app/src/constants/route.ts

```diff
@@ -1,10 +1,8 @@
 export const ROUTE = {
   ROOT: "/",
   GITHUB_CALLBACK: "/github/callback",
-
+  ROOM: "/room/:roomId",
```

Comment:
```suggestion
  ROOM: "/room",
```

when using the route, just do `${ROUTE.ROOM}/${roomId}`

Reply:
[2606523](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/18/commits/2606523af5b15ffff14fc5dbe04d832eaea49cde)